hazelcast-ra/hazelcast-jca/src/test/java/com/hazelcast/jca/ManagedConnectionFactoryImplTest.java

```diff
@@ -0,0 +1,89 @@
+package com.hazelcast.jca;
+
+import com.hazelcast.test.HazelcastParallelClassRunner;
+import com.hazelcast.test.HazelcastTestSupport;
+import com.hazelcast.test.annotation.QuickTest;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+
+import javax.resource.spi.ManagedConnectionFactory;
+
+import java.util.Arrays;
+import java.util.HashSet;
+
+import static org.junit.Assert.*;
+
+/**
+ * Created by mustafaiman on 25/06/15.
+ */
+@RunWith(HazelcastParallelClassRunner.class)
+@Category(QuickTest.class)
+public class ManagedConnectionFactoryImplTest extends HazelcastTestSupport {
+
+    ManagedConnectionFactory connectionFactory;
+
+    @Before
+    public void setup() throws Exception {
+        connectionFactory = new ManagedConnectionFactoryImpl();
+    }
+
+    @Test
+    public void testSetConnectionTracingEvents() {
+        ManagedConnectionFactoryImpl fact = (ManagedConnectionFactoryImpl)connectionFactory;
+        String controlStr = "DESTROY,  TX_START  , CLEANUP,TX_COMPLETE";
+
+        fact.setConnectionTracingEvents(controlStr);
+        String[] returnItems = delimitedStringToArray(fact.getConnectionTracingEvents(), ",");
+        String[] controlItems = delimitedStringToArray(controlStr,",");
+
+        Arrays.sort(controlItems);
+        Arrays.sort(returnItems);
+
+        assertArrayEquals(controlItems, returnItems);
+
+
+    }
+
+    @Test
+    public void testEquality() {
+        assertTrue(connectionFactory.equals(connectionFactory));
+        assertFalse(connectionFactory.equals(null));
+        assertFalse(connectionFactory.equals(new Object()));
+
+    }
+
+    @Test
+    public void testIdEqualityShouldDo() {
+        int id = 1234;
+        ManagedConnectionFactory c2 = new ManagedConnectionFactoryImpl();
+        ((ManagedConnectionFactoryImpl) connectionFactory).setId(id);
+        ((ManagedConnectionFactoryImpl) c2).setId(id);
+        assertTrue(c2.equals(connectionFactory));
+    }
+
+    @Test
+    public void testIdNotEqualityShouldReturnFalse() {
```

Comment:
what is the difference between this test and `testIdEqualityShouldDo` test.. seems they are same..